SWIG_CGAL/common.i

```diff
@@ -289,6 +289,17 @@
       SWIG_fail;
     }
 }
+
+%exception __next__
+{
+  try{
+      $action
+    }
+    catch(Stop_iteration){//TODO: throw a specify exception
+      SWIG_SetErrorObj(PyExc_StopIteration, SWIG_Py_Void());
+      SWIG_fail;
+    }
+}
```

Comment:
I'm wondering whether this is really needed since `__next()__` is calling `next()` that is already throwing.